pages/news/2020-XX-XX-new-in-2-3-rekordbox-support.html

```diff
@@ -0,0 +1,58 @@
+title: New in 2.3: Importing tracks and cues from Rekordbox removable devices
+author: ehendrikd
+date: 2020-07-13 20:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+
+Do you rock your gigs armed only with your cans and a Rekordbox prepared USB flash drive? Ever been tempted to play your carefully curated tracks on something other than a CDJ or Rekordbox, on something that is free and open-source? We have news for you!
+
+Mixxx 2.3 now supports reading the following from Rekordbox prepared removable devices on all of Mixxx's supported platforms:
```

Comment:
```suggestion
Mixxx 2.3 will support reading the following from Rekordbox prepared removable devices on all of Mixxx's supported platforms:

...

You can use this new feature now in [Mixxx 2.3 beta](/download/#unstable).
```

Reply:
Whoops, I didn't intend for this to be committed from the GitHub UI. Now the list has been deleted.